cmd/minishift/cmd/root.go

```diff
@@ -46,7 +48,7 @@ import (
 )
 
 var dirs = [...]string{
```

Comment:
@hferentschik Regarding initializing Viper and then checking profile does not seem to work for me. We can discuss about this.